apps/web/pages/haskolanam/um-haskolanam-is.tsx

```diff
@@ -0,0 +1,14 @@
+import withApollo from '@island.is/web/graphql/withApollo'
+import { withLocale } from '@island.is/web/i18n'
+import AboutPage from '@island.is/web/screens/UniversitySearch/AboutPage'
+import { getServerSidePropsWrapper } from '@island.is/web/utils/getServerSidePropsWrapper'
+
```

Comment:
is the '-is-' suffix meant to denote that this is the icelandic locale?

Reply:
No Its meant as "Um háskólanám.is"